src/main/webapp/app/complaints/form/complaints-form.component.scss

```diff
@@ -1,6 +0,0 @@
-.info-icon {
-    color: #ffc107;
-}
```

Comment:
Why is this class removed, is it defined at another place? It seems to be used in `complaints-form.component.html`.

Reply:
Good catch!

Reply:
Please verify that the icons for complaints and more feedback requests work correct